spec/requests/notification_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Notifications', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
@LYANGEND kindly remove this file , its not required for now

Reply:
Alright 👍